dusk-labyrinth.lic

```diff
@@ -45,19 +43,19 @@ class DuskLab
   def main(args)
     # search
     @pause_timer = Time.now
-    pause 1 until Flags['no-monkey'] || Flags['monkey'] || Flags['caught']
-    if Flags['no-monkey'] && Flags['five-minute-warning']
+    pause 1 until Flags['no-pet'] || Flags['pet'] || Flags['caught']
+    if Flags['no-pet'] && Flags['five-minute-warning']
       search
-      Flags.reset('no-monkey')
-    elsif Flags['no-monkey']
-      Flags.reset('no-monkey')
-    elsif Flags['monkey']
+      Flags.reset('no-pet')
+    elsif Flags['no-pet']
+      Flags.reset('no-pet')
+    elsif Flags['pet']
       search
-      Flags.reset('monkey')
+      Flags.reset('pet')
     elsif Flags['five-minute-warning'] || Flags['caught']
       search
     end
-    stow_loot(args.loot_bag)
+    stow_loot(args['loot_container'])
```

Comment:
What do you think of passing `@labyrinth_data['loot_container']` to `main(...)` instead of doing it this way?

Reply:
Sure, that makes a lot of sense. I was just using the `args` map that was already there. I don't know Ruby (but do know Nodejs) so I'm muddling my way through it, thanks for the suggestions :)

Changes made in https://github.com/rpherbig/dr-scripts/pull/4390/commits/201b9ab731dc4bfe3ff7e2384a80cdf5837774ac